cms/static/js/spec/views/group_configuration_spec.js

```diff
@@ -491,21 +491,6 @@ define([
             AjaxHelpers.expectNoRequests(requests);
         });
 
-        it('should have appropriate class names on focus/blur', function (done) {
```

Comment:
@dan-f The problem is that there is not a reliable way to test focus in the unit test. I tried various options when I was working on this PR, and I would find things that would work in "dev" mode but not "non-dev", and vice versa. But I couldn't find anything stable, and JQuery themselves have given up on this. I'll see if I can find some of the references I looked at a few months back...